includes/documents/abstract-wcpdf-order-document-methods.php

```diff
@@ -1184,7 +1184,38 @@ public function document_notes() {
 		}
 	}
 
+	
+	public function document_display_date() {
```

Comment:
Shouldn't this function code be inside the `get_display_date` function? Seems redundant to me having two similar functions.

Reply:
I followed same convention as it is used for other fields like notes and invoice number. I can move the code inside 'get_display_date'

Reply:
I see, in that case let's follow that then.